DBQnA/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -0,0 +1,178 @@
+
+For detailed information about these instance types, you can refer to this [link](https://aws.amazon.com/ec2/instance-types/m7i/). Once you've chosen the appropriate instance type, proceed with configuring your instance settings, including network configurations, security groups, and storage options.
+
+After launching your instance, you can connect to it using SSH (for Linux instances) or Remote Desktop Protocol (RDP) (for Windows instances). From there, you'll have full access to your Xeon server, allowing you to install, configure, and manage your applications as needed.
+
+## 🚀 Build Docker Images
+
+First of all, you need to build Docker Images locally. This step can be ignored once the Docker images are published to Docker hub.
+
+### 1.1 Build Text to SQL service Image
+
+```bash
+git clone https://github.com/opea-project/GenAIComps.git
+cd GenAIComps
+docker build --no-cache -t opea/texttosql:comps -f comps/texttosql/langchain/Dockerfile .
+
+```
+
+### 1.2 Build react UI Docker Image
+
+Build the frontend Docker image based on react framework via below command:
+
+```bash
+cd GenAIExamples/DBQnA/ui
+docker build --no-cache -t opea/texttosql-react-ui:latest -f docker/Dockerfile.react .
+
+```
+
+Then run the command `docker images`, you will have the following Docker Images:
+
+1. `opea/texttosql:latest`
+2. `opea/dbqna-react-ui:latest`
+
+## 🚀 Start Microservices
+
+### Required Models
+
+We set default model as "mistralai/Mistral-7B-Instruct-v0.3", change "LLM_MODEL_ID" in following Environment Variables setting if you want to use other models.
+
+If use gated models, you also need to provide [huggingface token](https://huggingface.co/docs/hub/security-tokens) to "HUGGINGFACEHUB_API_TOKEN" environment variable.
+
+### 2.1 Setup Environment Variables
+
+Since the `compose.yaml` will consume some environment variables, you need to setup them in advance as below.
+
+```bash
+# your_ip should be your external IP address, do not use localhost.
+export your_ip=$(hostname -I | awk '{print $1}')
+
+# Example: no_proxy="localhost,127.0.0.1,192.168.1.1"
+export no_proxy=${your_no_proxy},$your_ip
+
+# If you are in a proxy environment, also set the proxy-related environment variables:
+export http_proxy=${your_http_proxy}
+export https_proxy=${your_http_proxy}
+
+# Set other required variables
+
+export TGI_PORT=8008
+export TGI_LLM_ENDPOINT=http://${your_ip}:$TGI_PORT
+export HF_TOKEN=${HUGGINGFACEHUB_API_TOKEN}
+export LLM_MODEL_ID="mistralai/Mistral-7B-Instruct-v0.3"
+export POSTGRES_USER=postgres
+export POSTGRES_PASSWORD=testpwd
+export POSTGRES_DB=chinook
+export texttosql_port=9090
+```
+
+Note: Please replace with `your_ip` with your external IP address, do not use localhost.
+
+### 2.2 Start Microservice Docker Containers
+
+There are 2 options to start the microservice
+
+2.2.1 Start the microservice using docker compose
```

Comment:
Add heading to differentiate the header and content properly: 
#### 2.2.1 Start the microservice using docker compose